src/main/java/org/sagebionetworks/bridge/dynamodb/DynamoScanHelper.java

```diff
@@ -0,0 +1,13 @@
+package org.sagebionetworks.bridge.dynamodb;
+
+import java.util.Map;
+
+import com.amazonaws.services.dynamodbv2.document.Item;
+import com.amazonaws.services.dynamodbv2.document.api.ScanApi;
+
+public class DynamoScanHelper {
+    /** Scan a DDB object. */
+    public Iterable<Item> scan(ScanApi scannable, String filterExpression, String projectionExpression, Map<String, String> nameMap, Map<String, Object> valueMap) {
+        return scannable.scan(filterExpression, projectionExpression, nameMap, valueMap);
+    }
```

Comment:
I think we should use the no-args scan method and optimize for the common use case. Scan with filter almost never comes up.